package.json

```diff
@@ -1,35 +1,31 @@
 {
   "name": "synapse-oauth-signin",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
Updated SRC version, which means I have to update the way it builds (so it matches portals).